server/plugins/client/bundle.js

```diff
@@ -14,20 +28,27 @@ var parallel = require('async').parallel
  * and avoiding unneeded bundling with browserify saves a significant time.
  */
 function bundleClient (hoodieClientPath, bundleTargetPath, config, callback) {
-  parallel([
+  var plugins = [
+    path.resolve(config.paths && config.paths.applicationRoot || '', 'hoodie/client')
+  ].filter(checkModule)
+  var getPluginsModifiedTimes = plugins.map(function (pluginPath) {
+    return getModifiedTime.bind(null, requireResolve(pluginPath))
+  })
```

Comment:
`getPluginsModifiedTimes` sounds like a method, but it’s an array with values, right? Can we rename it to `pluginsModifiedTimes`?

If I understand it correctly, we get the entry JavaScript file for the app-specific client plugin, usually that would be `hoodie/client/index.js` or `hoodie/client.js`. 

The problem that we have is that while the client bundle will always be one file, the app-specific plugin (and 3rd party plugins in future) will most be spread around multiple files. I don’t have an idea how take this into account right now, I think this might end up being a very hard problem to solve :/ Maybe it’s something that browserify itself can do, with a custom plugin or something?

Hmmmmm 🤔